pkg/minikube/bootstrapper/kic/kic.go

```diff
@@ -0,0 +1,302 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kic
```

Comment:
How can we improve the bootstrapper interface and/or kubeadm enough to prevent the abstraction leakage  that makes it necessary for kic to have it's own kubeadm driver?

I'm sure it's the easiest for the initial implementation, but for production, I would rather see about improving our existing code than introducing a second subtly different kubeadm bootstrapper.